package.json

```diff
@@ -2,6 +2,9 @@
   "name": "index-app",
   "version": "2.0.0",
   "private": true,
+  "engines": {
+    "node": "20.x"
+  },
```

Comment:
Override vercel project setting - allows us to test versions on preview deployments before applying them project-wide. https://vercel.com/docs/functions/runtimes/node-js#version-overrides-in-package.json

Reply:
nice!